### PR TITLE
feat: build kubernetes-tools-kubectl image

### PR DESCRIPTION
**Pull request by sumo-drosiek**

### CODE REVIEW

File: Dockerfile.kubectl [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM alpine:3.17.3
```
Comment (swiatekm-sumo) [inline Dockerfile.kubectl:1]: This image doesn't have bash, and I think we want to have it even if it's not strictly necessary for the pvcCleaner script.

How about using the [official bash image](https://hub.docker.com/_/bash) as a base? That's also based on alpine, and scans come out clean on it.

File: Dockerfile.kubectl [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+ARG TARGETARCH
+ARG TARGETOS
+
+ENV KUBECTL_VERSION="v1.22.4"
+
+RUN apk update && apk add curl bash grep
```
[the remark this answers is not in the document]
Comment (sumo-drosiek) [reply Dockerfile.kubectl:8]: We don't need curl, we need grep

[the remark this answers is not in the document]
Comment (sumo-drosiek) [reply Dockerfile.kubectl:8]: > We should also make sure we clean all the apk cache here.

We are using scratch as an actual base

File: Dockerfile.kubectl (hunk 1 above)
Comment (sumo-drosiek) [reply Dockerfile.kubectl:1]: So do you propose to use `bash` instead of `scratch` or instead of `alpine` in this Dockerfile?

**Review by swiatekm-sumo — APPROVED** [review]

**Pull request merged**